src/spi/message/MessageClient.ts

```diff
@@ -101,7 +180,11 @@ export interface CommandReference {
     parameterName?: string;
 }
 
-export function buttonForCommand(buttonSpec: ButtonSpecification, command: any, parameters: any = {}): Action {
+export function buttonForCommand(buttonSpec: ButtonSpecification,
+                                 command: any,
+                                 parameters: {
+                                    [name: string]: string | number | boolean,
```

Comment:
What does `[name: string]` mean?

Reply:
It means parameters need a string key and can only be of string | number | boolean.

Reply:
Does `name` have any significance?

Reply:
Looks like not. https://www.typescriptlang.org/docs/handbook/interfaces.html#indexable-types